test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -305,6 +305,20 @@
           (is (= #{localhost-status test-agent-status revoked-agent-status}
                  (set (json/parse-string (:body response) true)))))
 
+        (testing "requires ignored path segment"
+          (let [response (test-app
+                          {:uri "/v1/certificate_statuses/"
+                           :request-method :get})]
+            (is (= 404 (:status response)))))
```

Comment:
It wasn't difficult to implement, and nice way get back in the swing of things after being sick, so I put up a PR at https://github.com/puppetlabs/puppet-server/pull/997 .